### DISCUSSION TITLE
How can I use static array data instead of Database Query

### DISCUSSION THREAD

**Discussion by lovecoding-git** [rappasoft/laravel-livewire-tables#1002] [Q&A]
How can I use static array data? like if I have 1000 rows array and I want to display it using this package. Thanks!

**Comment by lrljoe** [comment]
> How can I use static array data? like if I have 1000 rows array and I want to display it using this package. Thanks!

Not easily I'm afraid.  You could look at the builder function to get the relevant data into the table, referencing IDs from your array.
